### PR TITLE
add recent activites component

### LINKED ISSUE #32: Recent Activities Component
As per [Figma](https://www.figma.com/file/qOoG6pKbWKi2ldzHT2cHA6/Crypto-Component---Light-Mode-(Part-2)-(Community)?node-id=105%3A2514&t=zjwAxrDCZeGbfalZ-4)

### Preview

<div align="center">

Image

</div>

### Variants
- [ ] Light

### PR DESCRIPTION
**Pull request by NadiaFrShLm**
Fixes #32

Recent Activities Component #32
https://github.com/appwex/react-ui-components/issues/32#issue-1476617738

Screenshot

- add 'recent-activite' folder to routes folder
- add import line of the component to index.js
            import Activites from './routes/recent-activite/recent-activite' 
-add children element for const **router** to index.js
              {
        path: 'stories/recent-activite',
        element: <Activites />,
                },
-add link to Recent Activities Component at story-page.jsx
              <Link to={`/stories/recent-activite`}> Activites </Link>

### CODE REVIEW

File: src/routes/story-page/story-page.jsx [hunk 1]
```diff
@@ -12,6 +12,7 @@ const StoryPage = () => {
       <Link to={`/stories/notification`}>Notification Bar Component</Link>
       <Link to={`/stories/payment`}> Payment </Link>
       <Link to={`/stories/cookie`}> Cookie </Link>
+      <Link to={`/stories/recent-activite`}> Activites </Link>
```
Comment (aftabrehan) [inline src/routes/story-page/story-page.jsx:15]: Just a typo.
```suggestion
      <Link to={`/stories/recent-activities}> Activities </Link>
```

File: src/routes/recent-activite/recent-activite.jsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    url: 'https://images.unsplash.com/photo-1485893086445-ed75865251e0?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1480&q=80',
+  },
+  {
+    url: 'https://images.unsplash.com/photo-1584308972272-9e4e7685e80f?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1480&q=80',
+  },
+]
+
+const Activites = () => {
+  return (
+    <section className="activite">
```
Comment (aftabrehan) [inline src/routes/recent-activite/recent-activite.jsx:17]: Just a typo, please, make sure to update it in the styles sheet.

```suggestion
    <section className="activity">
```

File: src/index.js [hunk 3]
```diff
@@ -13,8 +13,9 @@ import ComponentStory from './routes/story-page/componet'
 import ButtonStory from './routes/story-page/button'
 import QuickTransfer from './components/quickTransfer/QuickTransfer'
 import NotificationStory from './routes/notification-page/index'
-import Payment from './routes/payment/payment'
+import Activites from './routes/recent-activite/recent-activite'
```
Comment (aftabrehan) [inline src/index.js:16]: Just a typo. Please, also make sure to update the component name, folder name, link, and path.

```suggestion
import Activities from './routes/recent-activity/recent-activity
```

**Review by aftabrehan — CHANGES_REQUESTED** [review]
Looks good to me! @NadiaFrShLm 💪 🚀 💯 

Just needs minor fixes and then it's good to go 🚀 

Could you please add a screenshot of your component in the PR description?

**Comment by mrxshinji** [comment]
Fixes #32

File: src/index.js (hunk 3 above)
Comment (NadiaFrShLm) [reply src/index.js]: - fixed the typos "Activites" -> "Activities" everywhere
- add blanc lines between selectors in css file
- completed the main post with screenshot and link to issue

**Review by aftabrehan — APPROVED** [review]
Excellent PR, @NadiaFrShLm 💪💯🚀

**Comment by aftabrehan** [comment]
> Excellent PR, @NadiaFrShLm 💪💯🚀

Resolve the conflicts @NadiaFrShLm @mrxshinji

**Pull request merged**